doc/nrf/libraries/caf/power_manager.rst

```diff
@@ -47,139 +47,140 @@ By default, the timeout is set to 120 seconds.
+Suspending application modules
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-A device reboot is required to exit this state.
+The |power_manager| submits :c:struct:`power_down_event`.
+While handling the event, application modules turn off the used peripherals or switch them to standby to conserve power.
+Only after all application modules confirmed that they have entered the low power state (by not consuming the :c:struct:`power_down_event`), the |power_manager| sets the required application power state.
+The |power_manager| registers itself as the final subscriber of the :c:struct:`power_down_event`.
+Reception of the :c:struct:`power_down_event` indicates that the |power_manager| can continue power down sequence.
+See :c:struct:`power_down_event` documentation for details regarding handling the event in application modules.
 
-Error
------
-
-The |power_manager| checks if any application modules have reported an error condition.
-
-When any application module switches to the error state (that is, broadcasts :c:enum:`MODULE_STATE_ERROR` through :c:struct:`module_state_event`), the |power_manager| puts the application into the error state.
-Then, after the period of time defined by :kconfig:option:`CONFIG_CAF_POWER_MANAGER_ERROR_TIMEOUT`, it puts the application to the off state.
-During this period, other modules can report the error condition to the user (for example, :ref:`caf_leds` can keep working in the error state).
-
-Restricting power states
-========================
-
-Any registered module can restrict the power state allowed by the usage of :c:struct:`power_manager_restrict_event`.
-It provides the module ID and the deepest allowed power state.
-The |power_manager| uses flags to restrict modes for any module.
-This means that you can repeatedly send the :c:struct:`power_manager_restrict_event`.
-
-Switching to low power
-======================
+Suspending application modules is done to reduce CPU workload and suspend the peripherals.
+It is assumed that the operating system will conserve power by setting the CPU state to idle whenever possible.
+If the system power management (:kconfig:option:`CONFIG_PM`) is enabled, reducing CPU workload also allows to enter deeper CPU sleep states.
 
-When the |power_manager| detects that the application is about to enter the low power state (either suspended or off), it sends a :c:struct:`power_down_event`.
-Other application modules react to this event by changing their configuration to low power, for example by turning off LEDs.
+System off state
+~~~~~~~~~~~~~~~~
 
-Some modules might not be ready to switch to the lower power state.
-In such case, the module that is not yet ready should consume the :c:struct:`power_down_event` and change its internal state, so that it enters the low power state when ready.
+After suspending the application modules, the |power_manager| can trigger entering the system off state (:c:func:`sys_poweroff`) to reduce the power consumption further.
```

Comment:
```suggestion
After suspending the application modules, the |power_manager| calls :c:func:`sys_poweroff`, so that it enters the system off state to reduce the power consumption further.
```
Please check the meaning.

Reply:
Rephrased a bit:
```
After suspending the application modules, the |power_manager| calls :c:func:`sys_poweroff`, so that system enters the off state to reduce the power consumption further.
```